### PR TITLE
UI Fixes & License

### PR DESCRIPTION
**Pull request by esunar**
Update all files to conform with Apache-2.0 License. 
Fixed help message capitalization and enhance help messages. 
Default values for parameters are explicitly shown.

### CODE REVIEW

File: cou/cli.py [hunk 1]
```diff
@@ -45,9 +44,13 @@ def parse_args(args: Any) -> argparse.Namespace:
     :rtype: argparse.Namespace
     """
     parser = argparse.ArgumentParser(
-        description="description",
+        description="Charmed OpenStack Upgrader(cou) is an application to upgrade Charmed "
+        "OpenStack. Application supports:\n  - Focal/Ussuri -> Focal/Victoria "
+        "\n  - Focal/Victoria -> Focal/Xena\n  - Focal/Xena -> Focal/Wallaby\n  - Focal/Wallaby "
+        "-> Focal/Yoga \nupgrades.",
```
Comment (rgildein) [inline cou/cli.py:50]: This is bad idea, since I can guarantee that you will forgot to update it when we support something new.

File: cou/cli.py [hunk 2]
```diff
@@ -61,17 +64,29 @@ def parse_args(args: Any) -> argparse.Namespace:
         dest="loglevel",
         choices=["DEBUG", "INFO", "WARNING", "ERROR", "CRITICAL"],
         type=str.upper,
-        help="Set the logging level",
+        help="Set the logging level. Defaults to INFO. This only affects stdout. The logfile "
```
Comment (rgildein) [inline cou/cli.py:67]: What is COU_DATA? Are we mention it somewhere in the help? Can we provide the actual path with f-string?

File: cou/cli.py (hunk 1 above)
Comment (esunar) [reply cou/cli.py]: I've moved supported releases to README.md

**Review by rgildein — APPROVED** [review]
LGTM

File: cou/cli.py [hunk 3]
```diff
@@ -45,9 +44,11 @@ def parse_args(args: Any) -> argparse.Namespace:
     :rtype: argparse.Namespace
     """
     parser = argparse.ArgumentParser(
-        description="description",
+        description="Charmed OpenStack Upgrader(cou) is an application to upgrade Charmed "
+        "OpenStack. Application supports N+1 upgrades.",
```
Comment (gabrielcocenza) [inline cou/cli.py:48]:
```suggestion
        "OpenStack. Application identify the lowest OpenStack version on the components and upgrade to the next version.",
```

File: cou/cli.py [hunk 4]
```diff
@@ -61,17 +62,29 @@ def parse_args(args: Any) -> argparse.Namespace:
...
     parser.add_argument(
         "--non-interactive", help="Run upgrade without prompt.", action="store_true", default=False
     )
+    parser.add_argument(
+        "-h",
+        "--help",
+        action="help",
+        default=argparse.SUPPRESS,
+        help="Show this help message and exit.",
+    )
```
Comment (gabrielcocenza) [inline cou/cli.py:87]: Is this necessary or you just want to be explicit on what `help` is doing?

**Review by gabrielcocenza — APPROVED** [review]
LGTM. I've left one comment and one suggestion that are no blockers

**Review by gabrielcocenza — APPROVED** [review]
LGTM

**Pull request merged**